lua/compe_nvim_lsp/source.lua

```diff
@@ -47,18 +49,32 @@ end
 
 function Source:complete(args)
   local params = vim.lsp.util.make_position_params()
+  local buf_has_clients = function()
```

Comment:
I think should be good to change to `if vim.lsp.client_is_stopped(self.client.id) then return args.abort() end`.